### DISCUSSION TITLE
Telecomando ARGO Silver Chic

### DISCUSSION THREAD

**Discussion by LucaGRETSCH** [indomus/forum#3238] [GENERALE - Discussioni generiche]
Salve sono un nuovo membro.
Ho visto in rete un progetto con BROADLINK per automatizzare un Termiventilatore ARGO Silver Chic.
Sono in possesso di un BROADLINK R4mini e lo utilizzo nella mio garage studio.
Nel bagno principale ho installato un ARGO Silver Chic, modello 191070155.. quindi non è l'ultimo modello di cui si trova il telecomando di ricambio.

Vedendo che dalla community sono disponibili i codici IR,  mi chiedevo se esistesse un telecomando programmabile che faccia uso di questi codici, senza dover prendere un altro broadlink.
Un saluto a tutti.

**Comment by KC-inDomus** [comment]
sai che non ho capito

**Reply by Scialla** [reply]
penso stia cercando un telecomando programmabile che possa acquisire i codici IR per avere un ricambio di scorta. Se così fosse, un qualsiasi telecomando universale in grado di acquisire le frequenze dal telecomando originale sarà sufficiente. Non credo esistano telecomandi in cui si possano inserire manualmente i codici delle frequenze IR che trovi online

**Reply by LucaGRETSCH** [reply]
scusate per la poca chiarezza.
Mi servirebbe un altro telecomando, perchè quello nativo è rotto e non comunica più. 
C'è difficoltà a reperirne uno originale per il mio modello. 

Pensavo di utilizzare un Broadlonk R4mini per caricare i codici del progetto inDomus, e poi magari con un telecomando programmabile copiare le IR dal Broadlink.
È fattibile? 
Che telecomando ci vorrebbe?

**Reply by Scialla** [reply]
in teoria dovrebbe essere possibile, dovresti cercare un telecomando universale per condizionatori in grado di apprendere i codici infrarossi.... la ricerca spetta a te però, anche perché qui link non si possono mettere.

Io però, prima di fare tutto sto casino, proverei a cercare un telecomando universale condizionatori già compatibile con la tua marca, ce ne sono con compatibilità al 99% con tutti i condizionatori.
Buona ricerca

**Reply by LucaGRETSCH** [reply]
Purtroppo l'assistenza ha definito il mio telecomando obsoleto. Tra l'altro quello in mio possesso non è funzionante.
L'unica opzione è provare ad usare la procedura col broadlink del progetto inDomus
Non so se sarò all'altezza, ma vale la pena provarci.
Possiedo già un Broadlink RM4 mini.
Mi toccherà tenerlo in bagno.

**Reply by Scialla** [reply]
Ma l'assistenza del tuo marchio? E mica devi cercarlo originale, guarda gli universali che sono compatibili con tutto, hai già guardato?
Vedi tu...

**Comment by LucaGRETSCH** [comment]
Si purtroppo l'assistenza Argo mi ha risposto così.

Esistono telecomandi universali per i condizionatori, ma per i
termoventilatori no.

Il mar 24 set 2024, 15:14 Scialla ***@***.***> ha scritto:
